bakery/src/scripts/utils.py

```diff
@@ -40,3 +45,10 @@ def parse_uri(uri):
         return None
     legacy_id, legacy_version = uri.split('@')
     return legacy_id, legacy_version
+
+
+def book_uuid_to_slug(book_uuid):
```

Comment:
If we plan to call this more than once during some step, do we care about doing the io here for the included file every time it is called?

Reply:
Good point. If we end up going in this direction, I can cleanup that aspect.